# ['sift', 'rootsift', 'tf1', 'tf2']
detector_type: tf2
# path to the saved model directory generated by exporting a model trained with the TensorFlow Object Detection API for TF2
saved_model: ~/.dodo_detector_ros/saved_model
# path to the frozen inference graphnfile generated by exporting a model trained with the TensorFlow Object Detection API for TF1
inference_graph: ~/.dodo_detector_ros/mscoco_frozen_inference_graph.pb
# path to TensorFlow Object Detection API label map
label_map: ~/.dodo_detector_ros/mscoco_label_map.pbtxt

# confidence level to report objects as detected
tf_confidence: 0.5

# minimum number of points to consider an object as present in the scene
sift_min_pts: 10

# path to the database used by the keypoint object detector
sift_database_path: ~/.dodo_detector_ros/sift_database

# the frame or tf that all object tfs will be published in relation to, leave blank to publish wrt. camera_link
# global_frame: map
